modules/core/src/merge.cpp

```diff
@@ -285,6 +346,16 @@ merge_( const T** src, T* dst, int len, int cn )
             int inc_i = (sizeof(T) == 8)? 1: 16/sizeof(T);
             int inc_j = 2 * inc_i;
 
+            VMerge2<T> vmerge;
+            for( ; i < len - inc_i; i += inc_i, j += inc_j)
+                vmerge(src0 + i, src1 + i, dst + j);
+        }
+#elif CV_VSX
```

Comment:
replace VMerge with universal intrinsics
```
#elif CV_VSX
if (hasSIMD128() && cn == 2)
{
    int inc_i = 16 / sizeof(T);
    int inc_j = 2 * inc_i;

    for (; i < len - inc_i; i += inc_i, j += inc_j)
        v_store_interleave(dst + j, v_load(src0 + i), v_load(src1 + i));
}
```
add int64 support for v_store_interleave in [intrin_vsx.hpp](https://github.com/justdoitqd/opencv/blob/20722a992215d46496fbf5abfed75dc03e18c4fa/modules/core/include/opencv2/core/hal/intrin_vsx.hpp#L334)
````
OPENCV_HAL_IMPL_VSX_INTERLEAVE(uint64, v_uint64x2)
OPENCV_HAL_IMPL_VSX_INTERLEAVE(int64, v_int64x2)
````